lerna.json

```diff
@@ -3,6 +3,7 @@
   "command": {
     "publish": {
       "ignoreChanges": [
+        "packages/package-json-lint/**",
```

Comment:
Should this be package-json-linter? instead of just lint?

Reply:
It's eslint and stylelint not eslinter or stylelinter.  I was trying to follow that pattern.